vtm-playground/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
     implementation project(':vtm-jts')
     implementation project(':vtm-mvt')
     implementation "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-desktop"
+    implementation 'org.mapsforge:mapsforge-map-awt:master-SNAPSHOT'
```

Comment:
We don't need a third-party dependency to have simple Java preferences, it's all inside the JDK.

_It's already wrong that Mapsforge hosts it inside the library, but late to change since it's public._

Reply:
Ok my fault